lib/requests/RequestDataTableColumns.js

```diff
@@ -4,7 +4,8 @@
 const RequestDataTableColumns = [
```

Comment:
maybe document this as well?  in particular:

- what does `data-table-expand` do?
- why do we use `text: ''` for some columns, and what are a11y requirements around that?